stories/mentions-multi-triggers/src/mentions.js

```diff
@@ -0,0 +1,48 @@
+const mentions = [
```

Comment:
Can you please change this file to a TypeScript file?

Reply:
@fxOne all `mentions` files are js file, as you can see.
![image](https://user-images.githubusercontent.com/51621949/108736823-f6247900-74ff-11eb-8cef-824d1d43a562.png)

Do you want me to change file to a typescript file for only multiTriggers?
Or do it for all?

Reply:
Would be nice if you have the time to migrate them all :)